ruby-challenges.rb

```diff
@@ -7,24 +7,96 @@
 
 filter_letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
-filter_letter_t = 't'
+# filter_letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def words_with_target_letter(words_array, target_letter)
+    filtered_words = []
+  
+    words_array.each do |word|
+      if word.include?(target_letter)
+        filtered_words.push(word)
+      end
+    end
+  
+    return filtered_words
+  end
+
+
+beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+
+filter_letter_o = 'o'
+result_o = words_with_target_letter(beverages_array, filter_letter_o)
+puts result_o.inspect
```

Comment:
Awesome job! When refactoring think about methods in ruby that will return arrays. Also keep in mind what the prompt says you custom method will take in.